lib/pbench/server/api/resources/server_configuration.py

```diff
@@ -0,0 +1,152 @@
+from http import HTTPStatus
+from logging import Logger
+
+from flask.json import jsonify
+from flask.wrappers import Request, Response
+
+from pbench.server import PbenchServerConfig
+from pbench.server.api.resources import (
+    API_AUTHORIZATION,
+    API_METHOD,
+    APIAbort,
+    API_OPERATION,
+    ApiBase,
```

Comment:
I would swap lines 11 & 12....

Reply:
Dealing with mixed case and punctuation is awkward here, and apparently `vscode` has its own opinions. 😆